www/sass/modules/page-elements/_search-drop-down.scss

```diff
@@ -54,7 +54,7 @@ div.common-branch-select {
 
 .select2-container {
 	min-width:90px;
```

Comment:
This file full of incorrect indentation, mixed two and four spaces, I would leave this fix for future